graphql-dgs-spring-boot-micrometer/src/main/kotlin/com/netflix/graphql/dgs/metrics/micrometer/DgsGraphQLMetricsInstrumentation.kt

```diff
@@ -177,18 +177,17 @@ class DgsGraphQLMetricsInstrumentation(
         }
     }
 
-    override fun beginExecutionStrategy(
-        parameters: InstrumentationExecutionStrategyParameters
```

Comment:
So switching to use InstrumenatationContext was teh issue?

Reply:
Exactly